subsys/net/l2/wifi/wifi_mgmt.c

```diff
@@ -813,6 +813,21 @@ static int wifi_get_connection_params(uint32_t mgmt_request, struct net_if *ifac
 
 NET_MGMT_REGISTER_REQUEST_HANDLER(NET_REQUEST_WIFI_CONN_PARAMS, wifi_get_connection_params);
 
+static int wifi_wps_config(uint32_t mgmt_request, struct net_if *iface, void *data, size_t len)
+{
+	const struct device *dev = net_if_get_device(iface);
+	const struct wifi_mgmt_ops *const wifi_mgmt_api = get_wifi_api(iface);
+	struct wifi_wps_pin_params *params = data;
+
```

Comment:
We could also have a check that verifies that len is sizeof(*params)

Reply:
I think we shouldn't check len is equal to sizeof(*params), because for WPS PBC case, len is 0